tokio/src/io/async_fd.rs

```diff
@@ -65,16 +65,16 @@ use std::{task::Context, task::Poll};
 /// # Examples
 ///
 /// This example shows how to turn [`std::net::TcpStream`] asynchronous using
-/// `AsyncFd`.  It implements `read` as an async fn, and `AsyncWrite` as a trait
-/// to show how to implement both approaches.
+/// `AsyncFd`.  It implements the read/write operations both as an `async fn`
+/// and using the IO traits.
```

Comment:
It might have just been because I was reading the diff, but for me it took a couple of reads of this line to realize what was going on. Perhaps calling out the names of the traits would make that easier.

Very much take it or leave it suggestion.

```suggestion
/// and using the IO traits `AsyncRead` and `AsyncWrite`.
```

Reply:
Done.